### PR TITLE
Fix #56

### LINKED ISSUE #56: Send emails to users when they did NOT read their messages.

### PR DESCRIPTION
**Pull request by Anduin2017**

### CODE REVIEW

File: Kahla.Server/Services/EmailNotifier.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    var dbContext = scope.ServiceProvider.GetRequiredService<KahlaDbContext>();
+                    var configuration = scope.ServiceProvider.GetRequiredService<IConfiguration>();
+                    var emailSender = scope.ServiceProvider.GetRequiredService<AiurEmailSender>();
+                    var users = await dbContext
+                                    .Users
+                                    .AsNoTracking()
+                                    .ToListAsync();
+                    foreach (var user in users.Where(t => t.EmailConfirmed))
+                    {
+                        int totalUnread = 0, inConversatons = 0, pendingRequests = 0;
```
Comment (ganlvtech) [inline Kahla.Server/Services/EmailNotifier.cs]: conversation typo

**Review by ganlvtech — CHANGES_REQUESTED** [review]

**Comment by ganlvtech** [comment]
Would it send the same email twice if I didn't read any message and no any new messages received?

**Review by Xiangrui2019 — COMMENTED** [review]
good!

**Review by xxyzz — APPROVED** [review]
I think the user should be able to enable or disable this feature.

**Comment by Anduin2017** [comment]
@xxyzz yes we will provide notification settings

**Review by ganlvtech — APPROVED** [review]

**Pull request merged**